Tasks/IISWebAppManagementOnMachineGroup/task.json

```diff
@@ -76,6 +76,14 @@
         }
     ],
     "inputs": [
+        {
+            "name": "EnableIIS",
```

Comment:
@kmkumaran is this task supposed to run on linux? it is PS based so I would assume no.